components/Tile/index.module.scss

```diff
@@ -0,0 +1,75 @@
+@import "styles/index";
+
+$shadow: 0 2px 22px 0 rgba(0, 0, 0, 0.15);
```

Comment:
Would this work being moved into the colors file, so other parts of MDL can make use of it?